src/helm/benchmark/metrics/metric.py

```diff
@@ -251,73 +224,70 @@ def derive_per_instance_stats(self, per_instance_stats: Dict[Instance, List[Stat
         """Derive stats based on existing per-instance stats, e.g., for calibration. Override me!"""
         return []
 
-    def compute_worst_case_metrics(self, per_instance_stats: Dict[Instance, List[Stat]]) -> List[Stat]:
```

Comment:
The only change I made to `compute_worst_case_metrics` was to move it out of the Metric class (e.g. dedent 1 scope). The diff viewer isn't making that very clear though.